### ISSUE TITLE
Can applications detect screen mirroring?

### ISSUE THREAD

**Issue by ThyZlayer** [Genymobile/scrcpy#2667]
for example I use scrcpy to screen mirror multiple netflix/ youtube premium/or any application with mobile then cast it to a monitor or multiple monitors for multiple people, it is not allowed and is against the rules, but question is could they detect it anyway?

I've tried researching and all I've seen is they can't detect it but they can disable the possibility of being mirrored.

Can applications really detect screen mirroring?, any possible ways applications could do so?

**Comment by roadSurfer** [comment]
Yes. Appx tyhat implement DRM will not display. See issue [4023.](https://github.com/Genymobile/scrcpy/issues/4023).
